src/server/exemption/activity-description/controller.test.js

```diff
@@ -74,10 +74,13 @@ describe('#activityDescriptionController', () => {
 
       expect(h.view).toHaveBeenCalledWith(ACTIVITY_DESCRIPTION_VIEW_ROUTE, {
         backLink: routes.TASK_LIST,
+        isMultiSiteJourney: false,
         isSiteDetailsFlow: false,
         pageTitle: 'Activity description',
         heading: 'Activity description',
-        payload: { activityDescription: undefined }
+        payload: { activityDescription: undefined },
+        projectName: undefined,
+        siteNumber: null
```

Comment:
why a mix of undefined and null, could we go with one or the other? 'null' is maybe a better fit for intentionally assigning no value?